File: app/src/main/java/pl/moozo/apicalc/data/remote/APIService.java
package pl.moozo.apicalc.data.remote;

import io.reactivex.Single;
import pl.moozo.apicalc.data.model.PostSum;
import retrofit2.http.Body;
import retrofit2.http.POST;

public interface APIService {

    @POST("/operations/sum")
    Single<String> sum(@Body PostSum postSum);
}

File: app/src/main/java/pl/moozo/apicalc/data/remote/APIService.kt
package pl.moozo.apicalc.data.remote

import io.reactivex.Single
import pl.moozo.apicalc.data.model.PostSum
import retrofit2.http.Body
import retrofit2.http.POST

interface APIService {

    @POST("/operations/sum")
    fun sum(@Body postSum: PostSum): Single<String>
}
